bindings/python/test/test_mongocrypt.py

```diff
@@ -370,21 +373,46 @@ def close(self):
 
 
 class TestMongoCryptCallback(unittest.TestCase):
+    maxDiff = None
 
     @staticmethod
     def mongo_crypt_opts():
         return MongoCryptOptions({
             'aws': {'accessKeyId': 'example', 'secretAccessKey': 'example'},
             'local': {'key': b'\x00'*96}})
 
+    def test_csfle(self):
+        # Test that we can pick up CSFLE automatically
+        encrypter = AutoEncrypter(MockCallback(), self.mongo_crypt_opts(),
+                                  bypass_encryption=False,
+                                  csfle_required=True,
+                                  )
```

Comment:
Please fix the code style issues:
```
                                  csfle_required=True,
                                  )
```

Should be:
```
                                  csfle_required=True)
```

Reply:
Done.